packages/application-components/src/components/dialogs/internals/dialog-footer.tsx

```diff
@@ -49,19 +50,32 @@ const DialogFooter = (props: Props) => {
         margin-top: ${uiKitDesignTokens.spacing50};
       `}
     >
-      <Spacings.Inline scale="m" alignItems="center" justifyContent="flex-end">
-        <SecondaryButton
-          label={getFormattedLabel(props.labelSecondary, intl)}
-          onClick={props.onCancel}
-          iconLeft={props.iconLeftSecondaryButton}
-          {...filterDataAttributes(props.dataAttributesSecondaryButton)}
-        />
-        <PrimaryButton
-          label={getFormattedLabel(props.labelPrimary, intl)}
-          onClick={props.onConfirm}
-          isDisabled={props.isPrimaryButtonDisabled}
-          {...filterDataAttributes(props.dataAttributesPrimaryButton)}
-        />
+      <Spacings.Inline alignItems="center" justifyContent="space-between">
+        <Spacings.Inline
+          scale="m"
+          alignItems="center"
+          justifyContent="flex-start"
+        >
+          <div>{props.footerContent}</div>
+        </Spacings.Inline>
```

Comment:
Unless there is a need for using `<Spacings.Inline>`, I would leave the layout style to consumers.

```suggestion
        <div>{props.footerContent}</div>
```

Reply:
updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3587/commits/046d7bbe7500a883dd38c7f1cb39c94014e8e5ea
Thank you.